classes/article/ArticleGalley.inc.php

```diff
@@ -218,6 +218,26 @@ function getFirstGalleyFile($fileType = null, $allFiles = false) {
 
 		return null;
 	}
+	
+	/**
+	 * Retrieve not the first, but all other revisions assigned to this submission file.
+	 * @param $fileId int
+	 * @return Array
+	 */
+	function getOtherRevisions($fileId) {
```

Comment:
It's better, if possible, to include some further qualification than just fetching the file by ID. The ID isn't checked against e.g. submission ID, so this could be abused to fetch files belonging to other submissions than intended.